subworkflows/local/assembly_qc.nf

```diff
@@ -84,10 +84,10 @@ workflow ASSEMBLY_QC {
     }
 
     PUBLISH_CLEANED_CONTIGS(
-        filtered_contigs
+        cleaned_contigs
     )
 
     emit:
-    filtered_contigs = filtered_contigs
+    filtered_contigs = cleaned_contigs
```

Comment:
We should also rename the output to avoid having to remember this name mapping outside the module

Reply:
To which output are you referring to, the one from this subwf or the one for published contigs? @mberacochea

Reply:
This one, in the emit:
```
filtered_contigs = cleaned_contigs
```
should be
```
cleaned_contigs = cleaned_contigs
```